docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,46 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+You can define custom `tsh` command aliases using the `aliases` section. It also allows for changing of the default flag values for existing commands.
+
+New command `tsh l`:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
+```
+
+Make `tsh status` use JSON as a default format:
+
+```yaml
+aliases:
+    "status": "tsh status --format=json"
+```
+
+Define custom command using with `bash`. The `$0` and `$1` variables will be substituted by command arguments.
+
+```yaml
+aliases:
+    "connect": "bash -c 'tsh login $0 && tsh ssh $1'"
+```
+
+The alias can use an arbitrary number of arguments. If a argument variable `$N` is referenced, `tsh` will check that at least `N+1` arguments were given to the alias invocation. All arguments that were given but not referenced in the alias definition will be appended at the end.
+
+Given configuration:
+
+```yaml
+aliases:
+    "example": "bash -c 'echo first=$0 $0-$1 $3'"
+```
+
+`tsh example 0 1 unused-2 3 unused-4` will expand to `bash -c 'echo first=0 0-1 3 unused-2 unused-4'`.
+
+To debug aliases, set the `TELEPORT_DEBUG=1` environment variable.
```

Comment:
More of an issue for the code and the docs, but it's a bit confusing now that teleport respects both the `DEBUG` and `TELEPORT_DEBUG` environment variables (and they do different things)

Reply:
Is this a new environment variable? I remember seeing it in the implementation PR but wasn't sure if it's new or not. If it's new, I agree it's confusing to have both.

Reply:
As far as I can tell, `tsh` did not respect `DEBUG` env var. Neither is new, both are next to each other:

https://github.com/gravitational/teleport/blob/24fcadc375d8359e80790b3ebeaa36bd8dd2822f/constants.go#L254-L266

We have had both `TELEPORT_DEBUG` and `DEBUG` for since 2016: 
https://github.com/gravitational/teleport/commit/21bd8caa4f31f05e0096cc94c25cf13ad91e394c#diff-25015a3bc3d26727834056ce5eb165af6fc8f8d5ee4142eb2a96872ec8fe501f

Between `TELEPORT_DEBUG` and `DEBUG` I think the first one is better: I can imagine the latter being defined for some unrelated reason.

Reply:
Gotcha, yeah, this is unfortunate. A couple of nitpicks here: I would replace "to debug" with "to troubleshoot", and also would expand this sentence to explain what setting this env var will actually do. Will it print logs in the stdout/err?

Reply:
I've changed the wording slightly, added an example and explained the expected outcome (logs in the console).